chapter_01.asciidoc

```diff
@@ -330,7 +336,7 @@ Next we can add the rest of the contents of the current folder, ".":
 ----
 $ pass:quotes[*git add .*]
 $ pass:quotes[*git status*]
-On branch master
+On branch main
```

Comment:
The default branch is still called `master` - at least on my fresh Linux Mint install in a VM in which I'm testing the book's instructions

Reply:
On my MacOS, it's main.

Reply:
Weird, on my Mac (v13.3.1) it's still `master`. I have git v2.40.0 which is just a minor behind the current latest. Maybe it's customized on yours? What do you get when you enter `git config --global init.defaultBranch` ?

Reply:
It says main, but I don't remember changing that. I really don't customize such things. Anyhow, if it's master for others, then let's keep master.